lib/metric-config-parser/metric_config_parser/data_source.py

```diff
@@ -77,6 +78,16 @@ class DataSource:
             `{dataset}` in from_expr if a value is not provided
             at runtime. Mandatory if from_expr contains a
             `{dataset}` parameter.
+        build_id_column (str, optional):
+            Default 'SAFE.SUBSTR(application.build_id, 0, 8)'.
+        friendly_name (str, optional)
+        description (str, optional)
+        joins (list[DataSourceJoin], optional)
+        columns_as_dimensions (bool, optional): Default false.
+        aggregation_units (list[AnalysisUnit], optional): denotes which
```

Comment:
I recall our discussion about `aggregation_unit` vs `analysis_unit`, but I'd assumed we'd be consistent: either `aggregation_unit/AggregationUnit` or `analysis_unit/AnalysisUnit`. Are we mixing names by calling the parameter `aggregation_unit` but the type `AnalysisUnit`?

Reply:
In my head there's sort of a distinction between aggregation unit (generic, what can we group by) and the analysis unit (specific to experimentation), but you're probably right. I'll go ahead with changing everything to `analysis_units`.